### PR TITLE
WIP: Add Bait set to sample when parsing orderform

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR adds Bait set on sample when parsinig a orderform.  I need help in defining a test for this PR.

**How to prepare for test**:
- [ ] ssh to hasta (depending on type of change)
- [ ] install on stage:
`bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

**How to test**:
- [ ] login to ...
- [ ] do ...

**Expected test outcome**:
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
I suggest that the test should verify the feature the PR adds to the system

File: cg/apps/lims/orderform.py [hunk 1]
```diff
@@ -191,6 +192,7 @@ def parse_sample(raw_sample):
     sample = {
         "application": raw_sample["UDF/Sequencing Analysis"],
         "capture_kit": raw_sample.get("UDF/Capture Library version"),
+        "bait_set": db.latest_bed_version(raw_sample.get("UDF/Capture Library version")).shortname,
```
Comment (patrikgrenfeldt) [inline cg/apps/lims/orderform.py:195]: I don't think this logic should go here

**Comment by patrikgrenfeldt** [comment]
This seems to need a description on how to test

File: cg/apps/lims/orderform.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/apps/lims/orderform.py:195]: this logic is for parsing samples from the orderform

Comment (moonso) [reply cg/apps/lims/orderform.py:195]: Let's look at this together when @mayabrandi is back on site

**Pull request closed without merging**